ingest-app/src/cmr/ingest/api/translation.clj

```diff
@@ -167,12 +167,11 @@
 
 (def random-metadata-routes
```

Comment:
This isn't for your pull request, but I am wondering why is this function in the code and not in test code.